storagenode/storagenodedb/database.go

```diff
@@ -225,17 +273,11 @@ func (db *DB) closeDatabases() error {
 
 // closeDatabase closes the specified SQLite database connections and removes them from the associated maps.
 func (db *DB) closeDatabase(dbName string) (err error) {
-	conn, ok := db.sqlDatabases[dbName]
+	mdb, ok := db.sqlDatabases[dbName]
 	if !ok {
-		return ErrDatabase.New("double close on database %s", dbName)
+		return ErrDatabase.New("no database with name %s", dbName)
```

Comment:
The only reason you could end up here is either 1. the database was never opened or there was a double close. So "no database with name" probably won't direct to the potential solution as well. But, these additional bits of information could be also in comment form, since when you look for this exact error message, you'll probably will find this location quite fast.

Reply:
Fixed! I made a better log message so it's more informative.